packages/cms/src/components/Viz/PercentageBar.jsx

```diff
@@ -16,7 +15,10 @@ class PercentageBar extends Component {
     const {dataFormat} = this.props;
 
     const defaults = {
-      cutoff: 5
+      cutoff: 5,
+      numberFormat: (d, value, total) => `${Number(d[value] / total * 100).toFixed(2)}%`,
```

Comment:
catch for `isNaN` here?

Reply:
I added the catch in both here and into the `reduce`, where I discovered that non-numerics can break my totaling logic.